src/fsharp/Fsc/Fsc.fsproj

```diff
@@ -30,23 +31,9 @@
       <InProject>false</InProject>
     </FilesToLocalize>
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="InternalsVisibleTo.fs" />
-    <Compile Include="..\..\assemblyinfo\assemblyinfo.fsc.exe.fs">
-      <Link>Resources/assemblyinfo.fsc.exe.fs</Link>
-    </Compile>
-    <Compile Include="..\fscmain.fs">
-      <Link>fscmain.fs</Link>
-    </Compile>
-    <None Include="fsc.exe.config">
-      <Link>fsc.exe.config</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\windows\default.win32manifest">
-      <Link>default.win32manifest</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
+
+  <Import Project="Fsc.Sources.proj" />
```

Comment:
Wowza.

> Anything built with the old asp.net template will need the legacy project system.

I can see if I can scrounge up some telemetry on this, but I highly doubt usage would be high enough to justify keeping it around just for that scenario.  Unless there is usage to prove otherwise, I think the stance here is that folks should upgrade to ASP.NET Core.